### PR TITLE
Adds all 4 confusion metrics

### PR DESCRIPTION
**Pull request by juliuscc**
Adds:

- accuracy
- recall
- precision
- F1-score

### CODE REVIEW

File: models/gradient-tree-booster.ipynb [hunk 1]
```diff
@@ -982,40 +984,41 @@
...
+      "\u001b[0;31mKeyboardInterrupt\u001b[0m: "
      ]
     }
    ],
    "source": [
     "from sklearn import ensemble\n",
+    "from sklearn.metrics import accuracy_score\n",
```
Comment (juliuscc) [inline models/gradient-tree-booster.ipynb:1018]: I do not know how to properly import multiple things from one module. Please suggest a fix to this.
